packages/web3-core-provider/src/index.ts

```diff
@@ -0,0 +1,57 @@
+import {
+    Web3Client,
+    Eip1193Provider,
+    IWeb3Provider,
+} from 'web3-core-types/lib/types';
+import Web3ProvidersHttp from 'web3-providers-http';
+import Web3ProvidersEip1193 from 'web3-providers-eip1193';
+
+import { ClientProtocol } from './types';
+
+export default function initWeb3Provider(
```

Comment:
could you add documentation comments of all functions

Reply:
yes, it's apart of the ToDo list above